tensorboard/plugins/profile/pod_viewer/details_card/details-card.html

```diff
@@ -0,0 +1,101 @@
+<!--
+@license
+Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<!--
+  details-card is a card that highlights detailed information of a selection.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-card/paper-card.html">
+
+<dom-module id="details-card">
+  <style>
+paper-card {
+  --paper-card-header-color: white;
+  --paper-card-header: {
+    background-color: rgb(178,34,34);
+  }
+  width: 100%;
+  max-height: 50vh;
+  overflow-y: auto;
+}
+.card-content > div {
+  top: -5px;
+  margin-bottom: 1em;
+}
+.value {
+  font-weight: normal;
+  text-align: right;
+}
+.info {
+  font-size: 20px;
+  font-weight: bold;
+}
+.codeStyle {
+  font-size: 14px;
+  font-weight: normal;
+}
+  </style>
+  <template>
+   <paper-card id="card" heading="[[name]] [[id]]"
+                hidden="[[!name]]" elevation="2">
+      <template is="dom-repeat" items=[[nodes]] as="node">
+        <div class="card-content info">
+          <div hidden="[[!isChannel]]">
+            <p>Replica Id: <span class="value">[[node.replicaId]]</span></p>
+          </div>
+          <div hidden="[[isStepBreakdown]]">
+            <p>Data Transferred: <span class="value">
+                [[sizeMiB_(node.dataSize)]] MiB</span></p>
+            <p>Latency: <span class="value">
+                [[format_(node.durationUs)]] Us</span></p>
+            <p>BW: <span class="value">
+                [[bw_(node.dataSize, node.durationUs)]] GiB/s</span></p>
+          </div>
+          <div hidden="[[!isChannel]]">
+            <p>Send Delay: <span class="value">
+                [[format_(node.sendDelayUs)]] Us</span></p>
+            <p>From: <span class="value">
+              Chip[[chipId_(node.srcCoreId)]], Core[[nodeId_(node.srcCoreId)]]
+            </span></p>
+            <p>To: <span class="value">
+              Chip[[chipId_(node.dstCoreId)]], Core[[nodeId_(node.dstCoreId)]]
+            </span></p>
+            <p>Hlo Names</p>
+            <code class="codeStyle">
+              <template is="dom-repeat" items=[[node.hloNames]]>"[[item]]"
+              </template>
+            </code>
+          </div>
+         <div hidden="[[!hasReplicaGroups]]">
+          <h5>Replica Groups</h5>
+          <code class="codeStyle">
+            <template is="dom-repeat" items=[[node.replicaGroups]]>
+             {[[item.replicaIds]]}<br>
```

Comment:
supernit: indentation is off. +1 whitespace

Reply:
Done.